roles/nginx/defaults/main.yml

```diff
@@ -1,5 +1,5 @@
 ---
-nginx_ppa: "ppa:nginx/mainline"
+nginx_ppa: "ppa:ondrej/nginx-mainline"
```

Comment:
Why not the official one?

See: https://www.nginx.com/resources/wiki/start/topics/tutorials/install/#official-debian-ubuntu-packages and https://www.ubuntuupdates.org/ppa/nginx_from_nginx.org?dist=focal

Reply:
Well we were using mainline (development) and `ppa:nginx/mainline` doesn't offer focal builds. Do the ones you listed support mainline? I couldn't see any mentions

Reply:
> If you would like to use mainline nginx packages, run the following command instead:
> 
> `echo "deb http://nginx.org/packages/mainline/ubuntu `lsb_release -cs` nginx" \
    | sudo tee /etc/apt/sources.list.d/nginx.list`
>
> — https://nginx.org/en/linux_packages.html

`focal` repo is under http://nginx.org/packages/mainline/ubuntu/dists/focal/

Note: 
- I haven’t tested it on Ubuntu 20.04
- the ansible task will need changes; similar to the mariadb one

Reply:
Oh awesome, thanks. Updated 😄